src/Catalyst.Cli.Tests/UnitTests/Commands/Response/GetPeerListResponseTests.cs

```diff
@@ -45,7 +45,7 @@ public void GetPeerListResponse_Can_Get_Output()
 
             var commandContext = TestCommandHelpers.GenerateCliResponseCommandContext(_testScheduler);
 
-            var getPeerListCommand = new PeerListCommand(commandContext, Substitute.For<ILogger>());
+            new PeerListCommand(commandContext, Substitute.For<ILogger>());
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Either remove this useless object instantiation of class 'PeerListCommand' or use it.](https://app.codacy.com/manual/catalyst-network/Catalyst.Framework/pullRequest?prid=4832783)